packages/xs-extra/vhd-tool.master/opam

```diff
@@ -25,12 +25,13 @@ depends: [
   "tar-unix"
   "uri"
   "vhd-format" {>= "0.7.0"}
+  "vhd-format-lwt" {>= "0.7.0"}
```

Comment:
`vhd-format-lwt` has been introduced starting at version 0.9.0, so we don't need this version bound, because it will always be true.